### PR TITLE
Product plan PR

### PR DESCRIPTION
**Pull request by zalmen**

### CODE REVIEW

File: product_plan.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+5. [Spacemesh Tap](tap.md) - Enables users to get `Spacemesh Coins` on testnets for testing purposes.
+
+6. [Spacemesh POST](https://github.com/spacemeshos/post) - Implement the proofs of space protocol used by Spacemesh.
+
+For additional information about these components read this page in the [Spacemesh Testnet guide](https://testnet.spacemesh.io/#/dict)
+
+---
+
+## Spacemesh 0.x
```
Comment (zalmen) [inline product_plan.md:54]: According to the table, this milestone will include "protocol updates" - which updates are included as part of this milestone?

File: product_plan.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+0.x releases are going to include bug fixes, updates and additional features for testing on the public testnet prior to mainanet launch. The mainnet will be launched once we are comfortable with the quality, security and performance of a 0.x release.
```
Comment (zalmen) [inline product_plan.md:55]: According to the table, this milestone contains "Tortoise self-healing" - is it co-ordinated with @tal-m ? When will the protocol is expected to be ready? If we want to deliver it by Q1/2020 the protocol must be delivered ASAP

File: product_plan.md (hunk 1 above)
Comment (avive) [reply product_plan.md:54]: All the protocol updates that will come before the mainent and will need to be testend in testnet - mostly the open research issues that the research team is currently on that will need to be implemented before mainent.

File: product_plan.md (hunk 2 above)
Comment (avive) [reply product_plan.md:55]: This is not one milestone - this is a series of 0.x releases. e.g. 0.2, 0.3, 0.4, ... etc - until we have tested on testnet all implemented features before mainnet. Sol, self-healing and any other feature will be included in a 0.x release - this is high level product plan does not detail in which specific 0.x release any feature will be delivered - it sets the framework and the terminology for these decisions.

File: product_plan.md (hunk 1 above)
Comment (zalmen) [reply product_plan.md:54]: There is a list of 5-7 protocol updates (for e.g, sub ids, poet discovery, genesis flow, etc.) that I'm aware of, that require implementation, plus 15 more updates planned for mainnet that are pending heavy research (AKA @iddo333 's list). According to the product plan, all of those need to be delivered by Q1 2020 (4 months after testnet launch). 

Aviv,
Where can I find a detailed list of product requirements including the protocol requirements for release 0.x? 

@tal-m @iddo333, 
When should the dev team expect the full protocol specifications that we want to use for mainnet?

File: product_plan.md (hunk 2 above)
Comment (zalmen) [reply product_plan.md:55]: According to the HL plan, all 0.x series need to be delivered during Q1 2020. If you want to include self-healing in 0.x I need to get the detailed protocol ASAP. When is it due? Where can I find the low-level product plan?

**Pull request closed without merging**